.github/workflows/tests.yaml

```diff
@@ -178,6 +191,9 @@ jobs:
           if [ -f ./logs/chromedriver.log ]; then cat ./logs/chromedriver.log; fi
           if [ -f ./logs/geckodriver.log ]; then cat ./logs/geckodriver.log; fi
           if [ -f ./logs/php-server.log ]; then cat ./logs/php-server.log; fi
+          if [ "$BROWSER_NAME" = "safari" ]; then
```

Comment:
Why not use just `-f` as with the other logs?

Reply:
Because I don't know the exact file.

```
❯ ls ~/Library/Logs/com.apple.WebDriver/safaridriver*
/Users/oandreyev/Library/Logs/com.apple.WebDriver/safaridriver.44337.qcntit.txt
/Users/oandreyev/Library/Logs/com.apple.WebDriver/safaridriver.45207.spaixu.txt
/Users/oandreyev/Library/Logs/com.apple.WebDriver/safaridriver.45618.mpzsjj.txt
/Users/oandreyev/Library/Logs/com.apple.WebDriver/safaridriver.46130.rzecqu.txt
/Users/oandreyev/Library/Logs/com.apple.WebDriver/safaridriver.52330.tijxyb.txt
~ ❯    
```